node/collation-generation/src/lib.rs

```diff
@@ -95,7 +96,7 @@ impl CollationGenerationSubsystem {
 				msg = receiver.next().fuse() => {
 					if let Some(msg) = msg {
 						if let Err(err) = ctx.send_message(msg).await {
-							log::warn!(target: LOG_TARGET, "failed to forward message to overseer: {:?}", err);
+							tracing::warn!(target: LOG_TARGET, err = ?err, "failed to forward message to overseer: {:?}", err);
```

Comment:
would we print error twice this way? I think the `{:?}", err` part should be removed
this applies to other places as well

Reply:
agree: 6ffc664